app/src/main/java/org/simple/clinic/util/RelativeTimestampGenerator.kt

```diff
@@ -40,6 +41,25 @@ class RelativeTimestampGenerator @Inject constructor() {
     val today = LocalDateTime.now(ZoneOffset.UTC)
     return generate(today, time)
   }
+
+  fun generate(instant: Instant, userClock: UserClock): RelativeTimestamp {
+    val then = instant.toLocalDateAtZone(userClock.zone)
```

Comment:
For me `day` has better readability than `then` in this entire function. Consider changing it?

Reply:
Sure.

Reply:
I renamed it to `date`.